compiler/src/iree/compiler/Codegen/Dialect/GPU/TargetUtils/KnownTargets.cpp

```diff
@@ -215,48 +235,147 @@ std::optional<TargetDetails> getAMDGPUTargetDetails(StringRef target) {
       .Case("rx7700xt", TargetDetails{rdna3Wgp, &rx7700xtChip})
       .Cases("rdna3", "gfx1100", "gfx1101", "gfx1102", "gfx1103", "gfx1150",
              "gfx1151", TargetDetails{rdna3Wgp, nullptr})
+      .Cases("rdna2", "gfx1030", "gfx1031", "gfx1032", "gfx1033", "gfx1034",
+             "gfx1035", "gfx1036", TargetDetails{rdna2Wgp, nullptr})
+      .Cases("rdna1", "gfx1010", "gfx1011", "gfx1012", "gfx1013",
+             TargetDetails{rdna1Wgp, nullptr})
       .Default(std::nullopt);
 }
 
 StringRef normalizeAMDGPUTarget(StringRef target) {
   if (target.starts_with("gfx"))
     return target;
 
+  // We cannot accept rdnaN as a target for LLVM AMDGPU backend; so the
+  // following is only meant for Vulkan but not HIP.
+  if (target.starts_with("rdna"))
+    return target;
+
   return llvm::StringSwitch<StringRef>(target.lower())
       .Case("mi300x", "gfx942")
       .Case("mi300a", "gfx940")
```

Comment:
BTW, isn't this off? I thought that the final HW for mi300a and x use the same target triple?

Reply:
hmm, okay I didn't know that.. Are you sure about this?

Reply:
We can change that separately in another PR